Makefile

```diff
@@ -65,6 +69,7 @@ quality:
 	./node_modules/jscs/bin/jscs $(STATIC_JS)/src --verbose
 	./scripts/run-pep8.sh
 	./scripts/run-pylint.sh
+	cd ./openassessment/ && i18n_tool validate
```

Comment:
Where is the `quality` target used?

Again, as mentioned in the other PRs, I feel this should be a part of the `validate_translations` target.

Reply:
i will add this in other pr. using this here just for testing will remove before merging.

Reply:
It's used in `tox.ini` here: https://github.com/edx/edx-ora2/blob/master/tox.ini#L27

`validate_translations` is called in that script as well, so I agree with the recommendation above.

Reply:
@amangano-edx  @efischer19 i will update other pr according to above suggestions.